### PR TITLE
Normalized stage_create_zip/create_zip

### PR DESCRIPTION
**Pull request by AhmedBM**
## Description

 * Normalized creation of MC packages using seperate stage_create_zip/create_zip for usage in signing MC packages

## Checklist

- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-osconfig/blob/main/CONTRIBUTING.md).
- [X] All unit tests are passing.
- [X] I have merged the latest `main` branch prior to this PR submission.
- [X] I submitted this PR against the `main` branch.

### CODE REVIEW

**Review by simonjaeger — APPROVED** [review]

File: src/adapters/mc/asb/CMakeLists.txt [hunk 1]
```diff
@@ -23,15 +23,15 @@ set_target_properties(OsConfigResourceAsb PROPERTIES OUTPUT_NAME libOsConfigReso
 
 # Create the policy artifacts ZIP package for the Azure Security Baseline for Linux
 
-add_custom_target(stage_create_zip
+add_custom_target(stage_create_zip_asb
     COMMAND ${CMAKE_COMMAND} -E make_directory ${PROJECT_BINARY_DIR}/Staging
```
Comment (MariusNi) [inline src/adapters/mc/asb/CMakeLists.txt:27]: Do we also need mc/ssh one to be changed or only this one to make the two different?

File: src/adapters/mc/asb/CMakeLists.txt [hunk 2]
```diff
@@ -23,15 +23,15 @@ set_target_properties(OsConfigResourceAsb PROPERTIES OUTPUT_NAME libOsConfigReso
...
     COMMAND ${CMAKE_COMMAND} -E copy_if_different "${CMAKE_CURRENT_SOURCE_DIR}/AzureLinuxBaseline.mof" ${PROJECT_BINARY_DIR}/Staging/
     COMMAND ${CMAKE_COMMAND} -E copy_if_different $<TARGET_FILE:OsConfigResourceAsb> ${PROJECT_BINARY_DIR}/Staging/Modules/DscNativeResources/OsConfigResource/libOsConfigResource.so
     DEPENDS OsConfigResourceAsb)
 
-add_custom_target(create_zip ALL
+add_custom_target(create_zip_asb ALL
     BYPRODUCTS ${OsConfigRootBinaryDir}/AzureLinuxBaseline.zip
     COMMAND ${CMAKE_COMMAND} -E tar "cfv" "${OsConfigRootBinaryDir}/AzureLinuxBaseline.zip" --format=zip .
-    DEPENDS stage_create_zip
+    DEPENDS stage_create_zip_asb
```
Comment (MariusNi) [inline src/adapters/mc/asb/CMakeLists.txt:36]: >stage_create_zip_asb [](http://example.com/codeflow?start=12&length=20)

nit: stage_create_asb_zip looks better (zip kept at the end as it what's gets created)

File: src/adapters/mc/asb/CMakeLists.txt (hunk 1 above)
Comment (AhmedBM) [reply src/adapters/mc/asb/CMakeLists.txt:27]: just this one

Comment (MariusNi) [reply src/adapters/mc/asb/CMakeLists.txt:27]: Let's also rename that one state_create_ssh_zip to keep consistent. Thanks

**Review by MariusNi — APPROVED** [review]
:shipit:

**Pull request merged**